src/components/exercises/ExerciseItem.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom';
+
+import { ReactComponent as EditIcon } from '../../icons/edit.svg';
+import { ReactComponent as DeleteIcon } from '../../icons/delete.svg';
+
+import classNames from 'classnames';
+import { Exercise } from '../../types';
+
+import './ExerciseItem.css';
+
+const ExerciseItem: FC<{
+    exercise: Exercise;
+    onDelete: () => void;
+    className: string;
+}> = ({ exercise, onDelete, className }) => {
+	return (
+		<div className={classNames("ExerciseItem", className)}>
+      <img className="image" src={exercise.image} alt="Upload"/>
```

Comment:
weird formatting. Do you have an autoformatter / linter that you could apply?

Reply:
It's fixed

Reply:
I don't see an update in the formatting here. Was it fixed in another branch?